pkg/kubectl/cmd/util/helpers.go

```diff
@@ -45,7 +46,8 @@ import (
 )
 
 const (
-	ApplyAnnotationsFlag = "save-config"
+	ApplyAnnotationsFlag      = "save-config"
+	ChangeSourceAnnotationKey = "kubectl.kubernetes.io/change-source"
```

Comment:
I agree with this. I also think the contents should effectively be considered opaque - I was hoping to sneak the git hash of the commit that kubectl apply was run on into this annotation as well.